skins/cat17/templates/Donation_Comment.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'Base_Layout.html.twig' %}
+
+{% block page_identifier %}donation-comment{% endblock %}
+
+{% block main %}
+	<main class="comment">
```

Comment:
You need to remove the `main`element, it's already in `Base_Layout.twig`. This will get rid of the blank space you mentioned in https://phabricator.wikimedia.org/T182126#3829622